storage/rocksdb/ha_rocksdb.h

```diff
@@ -360,6 +372,8 @@ struct st_global_stats {
   ib_counter_t<ulonglong, 64, RDB_INDEXER> system_rows[ROWS_MAX];
 
   ib_counter_t<ulonglong, 64, RDB_INDEXER> queries[QUERIES_MAX];
+
+  ib_counter_t<ulonglong> covered_secondary_key_lookups;
```

Comment:
IIRC that was one of recent upstream performance regressions, which has been fixed later, would be good to merge it by the next release at least

Reply:
Yes, good eye. I actually meant to pull that commit into this August series (it is in next few commits in September series) and forgot about it. (69512214a5d9b0888e20c8cd369c1fde996674b2 Fixing performance regression because of using ib_counter_t incorrectly) I'll go drag it into this series and update the PR.